README.md

```diff
@@ -1,29 +1,20 @@
 # Azure SDK for Rust
```

Comment:
Not a blocker, but I have to say that "SDK" instead of "SDKs" has always bugged me a bit. All our languages release multiple packages. Those are separate "SDKs". There's the .NET SDK, the .NET EF SDK, the MAUI SDK, etc., for .NET. Are we sure we're not the "Azure SDKs for Rust"?